doc/rho.1

```diff
@@ -0,0 +1,408 @@
+.IP
+The kernel for the server.
+.PP
+processor
+.IP
+The processor type for the server.
+.PP
+platform
+.IP
+The hardware platform for the server.
+.PP
+release name
+.IP
+The type of release. For Red Hat Linux, this is redhat-release. For up- and downstream releases, this could be centos-release or fedora-release. The formats for other Unix and Linux distributions depends on the distribution.
+.PP
+release version
+.IP
+The major version of the release.
+.PP
+release number
+.IP
+The release number for the operating system release.
+.PP
+system ID
+.IP
+The identifier for the server, as in /etc/sysconfig/rhn/systemid.
+.I For Red Hat releases only.
+.PP
+username
+.IP
+The username used by the server to bind to Red Hat Network for updates, as in /etc/sysconfig/rhn/systemid.
+.I For Red Hat releases only.
+.PP
+instnum
+.IP
+The instance number for the server for entitlement management, as in /etc/sysconfig/rhn/systemid.
+.I For Red Hat releases only.
+.PP
+release
+.IP
+The full release name, as in the /etc/*release file.
+.I Mainly for operating systems other than Red Hat Linux.
+.PP
+CPU count
+.IP
+The number of CPUs on the machine.
+.PP
+CPU vendor
+.IP
+The name of the CPU manufacturer, from the /proc/cpuinfo file.
+.PP
+BIOS vendor
+.IP
+The name of the BIOS vendor, from running dmidecode -s bios-vendor.
+.PP
+virtual guest/host
+.IP
+The type of virtual machine. This can be baremetal (no virtual machines), host, or guest.
+.PP
+virtual type
+.IP
+The type of virtual service. This can be xen, qemu, kvm, or vmware.
+.PP
+auth type
+.IP
+The SSH authentication method used by rho. This can be ssh (simple password) or ssh_key.
+.PP
+auth username
+.IP
+The username used by rho to bind to the system.
+.PP
+auth name
+.IP
+The name of the auth profile used by rho for the scan.
+.PP
+error
+.IP
+Any errors returned by the SSH process. This is normally "login failed" or "unable to connect".
+
+.SH EXAMPLES
+.PP
+.TP
+Adding new auth credentials
+rho auth add --name=new-creds --username rho-user --sshkeyfile /etc/ssh/ssh_host_rsa_key
+.PP
+.TP
+Creating a new profile
+rho profile add --name=new-profile --hosts 1.2.3.0 --auth new-creds
+.PP
+.TP
+Editing a profile
+rho profile edit --name new-profile --range "1.2.3.0 - 1.2.3.255" --auth new-creds other-creds
+.PP
+.TP
+Running a scan with a profile
+rho scan --profile new-profile --reportfile=/home/jsmith/Desktop/output.csv
+
+.SH SECURITY CONSIDERATIONS
+The credentials used to access servers are stored with the profile configuration in an AES-128 encrypted configuration file (rho.conf). A passphrase is used to access this file. The passphrase and decrypted file contents are in the system memory, and could theoretically be written to disk if they were to be swapped out.
+.PP
+While the password-related environment variables can be set to run 'rho' without prompts (such as scheduling a cron job), setting these variables is extremely risky. Be exceptionally cautious about using these variables.
```

Comment:
I would rather remove this paragraph, or at least tone down the caution some. I don't think users need to be "exceptionally cautious", just "cautious".